flaml/automl.py

```diff
@@ -906,6 +913,8 @@ def custom_metric(X_test, y_test, estimator, labels,
             self._state.n_jobs = n_jobs
             self._search()
             logger.info("fit succeeded")
+        if verbose:

```

Comment:
what if verbose==0?

Reply:
my bad, this should be "if verbose==0" (so that if we have previously changed logger level, we revert it to what it was before.